docs/guides/software-lifecycle/security/github-security/README.md

```diff
@@ -48,14 +48,19 @@ If you do not have organizational permissions or if you wish to customize securi
      - For added security, we suggest turning on Dependabot security updates to automatically generate pull requests for known vulnerabilities in your dependencies.
      - We also recommend enabling Dependabot version updates _if you are using a package manager for your project_. This will help you keep your dependencies up-to-date. To configure Dependabot version updates:
        1. Create a `.github/dependabot.yml` file in your repository.
-       2. Specify the package-ecosystem, directory, and schedule for the updates. For example:
+       2. Specify the package-ecosystem, directory, schedule and branch to update. For example (from the [Slim Python Starter Kit](https://github.com/NASA-AMMOS/slim-starterkit-python/blob/main/.github/dependabot.yml)):
           ```yml
           version: 2
           updates:
-            - package-ecosystem: "npm"
-              directory: "/"
+            - package-ecosystem: 'pip'
```

Comment:
Explanation of changes: 
1. Tie back to Python Starter Kit
2. Add slightly more detail to configuration for a very routine request to control analysis time.
3. Limited comments to explain.

Reply:
Nice @ingyhere - much more useful to tie back to other SLIM projects and to do this for Python. Thanks!